companies/box.md

```diff
@@ -0,0 +1,12 @@
+---
+fortune_rank: 93
```

Comment:
I see ConocoPhillips as Fortune ranking 93.  Where'd you find this data?

![image](https://user-images.githubusercontent.com/21069635/97476383-54f43000-191c-11eb-83d9-e6265d1c11b0.png)

Reply:
From https://fortune.com/fortune500/2020/search/